Samples/Flix/src/main/java/dev/testify/samples/flix/domain/model/HomeScreenDomainModel.kt

```diff
@@ -0,0 +1,7 @@
+package dev.testify.samples.flix.domain.model
+
+data class HomeScreenDomainModel(
```

Comment:
This definitely comes down to a preference thing, but I like keeping my class names as short as possible and rely on the packages for disambiguation. So, for these classes, I feel like `HomeScreen` is a sufficiently descriptive name that we could drop the `DomainModel` suffix.

Reply:
I'll think about that. It's quite likely that one would end up with multiple classes having the same name which isn't the best either.